### ISSUE TITLE
Problem using the library

### ISSUE THREAD

**Issue by damianocosco** [wintercms/wn-location-plugin#6]
Good morning, 
I installed this plugin, but I found a problem. If there is a relationship in the model I use it in, the popups to manage relationships are not opened and remain loading. I noticed in the console that when I click on the relationship records it tries to load a js in an incorrect Google path. 
Is this a known library issue? 
Thank you

**Comment by LukeTowers** [comment]
@damianocosco no, I haven't seen that before. Could you provide more details with the exact error?

**Comment by LukeTowers** [comment]
@damianocosco I've just run into this myself on a project and fixed it in v2.2.2. Related: wintercms/winter#1538
